docs/install/thetagpu.rst

```diff
@@ -8,27 +8,38 @@ ThetaGPU (Argonne LCF)
 Already installed module
 ========================
 
-This installation procedure shows you how to access the installed DeepHyper module on ThetaGPU. After logging in Theta, connect to a ThetaGPU service node:
+This installation procedure shows you how to access the installed DeepHyper module on ThetaGPU. It may be useful to wrap these commands in this ``activate-dhenv.sh`` script :
```

Comment:
Let's try to be more consistent and use the same name for the activation script everywhere, I think it is `init-dh-environment.sh`